uk.co.compucorp.civicrm.hrcomments/tests/phpunit/CRM/HRComments/BAO/CommentTest.php

```diff
@@ -96,7 +96,7 @@ public function testTryingToSoftDeleteCommentOnUpdateThrowsException() {
     ]);
   }
 
-  public function testCreatedDateIsInsertedCorrectly() {
+  public function testCreatedDateIsInsertedCorrectlyWhenNotPassedAsAParameter() {
```

Comment:
please rename this test to something more specific like `testCreatedDateIsInsertedAsTheCurrentDateWhenNotPassedAsAParameter`